pkgs/servers/seafile-server/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, lib, fetchFromGitHub, pkg-config, python3Packages, autoreconfHook
+, libuuid, sqlite, glib, libevent, libsearpc, openssl, fuse, libarchive, which
+, vala, cmake, oniguruma }:
+
+let
+  libevhtp = import ./libevhtp.nix {
+    inherit stdenv lib fetchFromGitHub cmake libevent;
+  };
```

Comment:
Why is this not in global?

Reply:
You mean declared as a public nix package? Because seafile-server relies on a non-upstream fork for libevhtp containing specific patches.
This fork is out of sync with upstream so we don't want any other package to rely on this specific libevhtp package.

Reply:
A comment with that would be nice.